From: =?utf-8?b?SMOpY3RvciBPcsOzbiBNYXJ0w61uZXo=?= <zumbi@debian.org>
Date: Mon, 5 Mar 2018 01:16:37 +0100
Subject: deb: don't create an empty rcS
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: base64

cmNTIGlzIGEgY29uZmlndXJhdGlvbiBmaWxlIGNyZWF0ZWQgYnkgaW5pdHNjcmlwdHMsIG5vdCBl
bnRpcmVseSBjbGVhcgp3aHkgb2JzLWJ1aWxkIHRyaWVzIHRvIGNyZWF0ZSBpdC4gQnV0IGlmIGl0
IGRvZXMgaW5pdHNjcmlwdHMgd2lsbCBhc2sKd2hldGhlciB0byBrZWVwL3JlcGxhY2UgaXQgaW50
ZXJhY3RpdmVseSBibG9ja2luZyB0aGUgYnVpbGQuCgpTaWduZWQtb2ZmLWJ5OiBTam9lcmQgU2lt
b25zIDxzam9lcmQuc2ltb25zQGNvbGxhYm9yYS5jby51az4KU2lnbmVkLW9mZi1ieTogSMOpY3Rv
ciBPcsOzbiBNYXJ0w61uZXogPHp1bWJpQGRlYmlhbi5vcmc+Cg==
---
 build-pkg-deb | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build-pkg-deb b/build-pkg-deb
index ea2fc2a..c5959c0 100644
--- a/build-pkg-deb
+++ b/build-pkg-deb
@@ -48,7 +48,6 @@ deb_setup() {
     :>> $BUILD_ROOT/var/lib/dpkg/available
     :>> $BUILD_ROOT/var/log/dpkg.log
     :>> $BUILD_ROOT/etc/ld.so.conf
-    :>> $BUILD_ROOT/etc/default/rcS
 }
 
 pkg_initdb_deb() {
